injector/network_disruption_test.go

```diff
@@ -358,6 +366,50 @@ var _ = Describe("Failure", func() {
 
 		})
 
+		Context("with a service and allowed ports specified", func() {
+			BeforeEach(func() {
+				spec.Services = []v1beta1.NetworkDisruptionServiceSpec{
+					{
+						Name:      "foo",
+						Namespace: "bar",
+						Ports:     []int{80},
+					},
+				}
+
+				podsWatcher := watch.NewFake()
+				servicesWatcher := watch.NewFake()
+
+				k8sClient.PrependWatchReactor("pods", testing.DefaultWatchReactor(podsWatcher, nil))
+				k8sClient.PrependWatchReactor("services", testing.DefaultWatchReactor(servicesWatcher, nil))
+
+				// fake watchers for service handling
+				go func() {
+					// Set up
+					time.Sleep(300 * time.Millisecond)
```

Comment:
Do we really need to add this two sleeps?
_IMO: Sleep statements in unit tests can significantly slow down the execution time, which can be inconvenient and difficult to work with._